src/geometry/real.rs

```diff
@@ -119,6 +120,13 @@ mod real_impl {
             src.0.to_num::<u32>()
         }
     }
+
+    #[cfg(feature = "defmt")]
+    impl ::defmt::Format for Real {
+        fn format(&self, fmt: ::defmt::Formatter<'_>) {
+            ::defmt::write!(fmt, "Read({=i32:X})", self.0.to_bits())
```

Comment:
The hex value is hard to decipher. Also printing the approximate value by converting into floating point would make the output more useful IMO. Something like this might work:
```suggestion
            ::defmt::write!(fmt, "Real({=i32:#X} ≈{=f32})", self.0.to_bits(), self.0.to_num())
```

Reply:
Good idea! I didn't use display/debug methods, as the whole point of defmt is to not pull those in, but hadn't considered converting it to a float.